src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/NestedForDepth/context/expected.txt

```diff
@@ -0,0 +1,4 @@
+Test.java:17:17: Nested for depth is 2 (max allowed is 1).
```

Comment:
can you sort expected violations by line number?

Reply:
@baratali now we can not sort expected violations by line number, this problem should be solved later.

Reply:
@HuGanghui , why it can't be sorted? (do you have a github issue to fix this problem?)

Reply:
@baratali what I mean is that probably it is not easy to be solved, I will make a issue for it, and try to solve it after I finish all prs of checks listed in https://github.com/checkstyle/patch-filters/issues/100, I'll probably start working on it tomorrow.